app/src/main/java/fr/free/nrw/commons/campaigns/CampaignsPresenter.java

```diff
@@ -108,7 +107,7 @@ public void getCampaigns() {
                     }
 
                     @Override public void onError(Throwable e) {
-                        Log.e(TAG, "could not fetch campaigns: " + e.getMessage());
+                        Timber.e(e.getMessage(), "could not fetch campaigns");
```

Comment:
In this form the first argument needs to be Throwable. Just `Timber.e(e, "could not fetch campaigns")` will do the job, I believe.